pkg/operator/ceph/cluster/mgr/spec.go

```diff
@@ -202,6 +203,15 @@ func (c *Cluster) makeMgrDaemonContainer(mgrConfig *mgrConfig) v1.Container {
 			c.cephMgrOrchestratorModuleEnvs()...,
 		),
 		Resources: c.resources,
+		LivenessProbe: &v1.Probe{
```

Comment:
Could we add a liveness probe for both the dashboard and the prometheus modules? Hopefully the self-signed cert on mimic and newer won't cause problems.

Reply:
Since modules can be enabled/disabled on the fly, the liveness check could fail for that. I believe sticking with `/` on `metricsPort` is already good enough.